op-e2e/system_test.go

```diff
@@ -211,6 +291,114 @@ func TestValidationReward(t *testing.T) {
 	}
 }
 
+func TestValidatorSystemUpgrade(t *testing.T) {
```

Comment:
How about setting the trusted validator for v1 and v2 to different accounts to make sure the validator system is upgraded well? And after termination index, it would be good to make sure the output submitter is the trusted validator for v2. And it would be better if the test name is `TestValidatorSystemUpgradeToV2`.

Reply:
I want to check that a validator who is eligible for validator system v2 submit output right after the exact termination index of output.

Reply:
Can you explain it once more? Isn't it enough to check the reward system in `ValidatorManager` works well after termination?

Reply:
It would be nice to see if the upgrade go through at the correct point in time (termination output index + 1) by checking if the validator is eligible for v2.
I agree to set the same trusted validator for both version because it is trusted validator. But I have no idea to check this. If you think it's ok, let's leave it as is for now.

Reply:
I added a check of the output index in first reward distribution event in following PR https://github.com/kroma-network/kroma/pull/325.